ZhuoHeiChaUI/Events/InitializeGameStateEventArgs.cs

```diff
@@ -9,6 +9,6 @@ public class InitializeGameStateEventArgs : EventArgs
     {
         public List<int> cardBefore;
         public List<int> cardAfter;
```

Comment:
As stated before, please follow the C# naming conventions and use `PascalCase` for public-facing members.

Reply:
Done